static/app/views/settings/organizationMembers/organizationMembersList.spec.tsx

```diff
@@ -42,6 +42,39 @@ const roles = [
   },
 ];
 
+const missingMembers = [
+  {
+    integration: 'github',
+    users: [
+      {
+        commitCount: 6,
+        email: 'hello@sentry.io',
```

Comment:
Should we test with emails not ending with `@sentry.io`? Probably will not cause problems but you never know

Reply:
this is a mock API response so i don't think it makes a difference